packages/vm/src/testing/contract.rs

```diff
@@ -95,15 +99,30 @@ impl<'a> Contract<'a> {
     pub fn change_options(&mut self, options: MockInstanceOptions<'a>) {
         self.options = options
     }
+
+    /// update balance in backend querier. it does not change the options
+    pub fn update_balance<T: Into<String>>(&mut self, addr: T, balance: Vec<Coin>) -> TestingResult<Option<Vec<Coin>>> {
+        let mut backend = self
+            .backend
+            .take()
```

Comment:
Do we have to 'take'? how about just use to 'match' and not assigning the backend again?

Reply:
Thanks! reflected!